### ISSUE TITLE
should aria-hidden subtrees be ignored by clickButton, fillIn, etc?

### ISSUE THREAD

**Issue by avh4** [avh4/elm-program-test#156]
```html
<body>
    <div aria-hidden=true>
        <button>A</button>
    </div>
</body>
```

```elm
ProgramTest.clickButton "A"
```

Should the clickButton fail because its inside an `aria-hidden=true` element?  I think that doesn't really make sense because isn't the button still visible on the page (even though hidden from accessibility tools) unless there is also CSS applied to hide it visually?
